libs/workspace/src/builders/barista-build/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://json-schema.org/draft-07/schema",
+  "title": "Workspace builder",
+  "description": "",
+  "type": "object",
+  "properties": {
+    "outputPath": {
+      "type": "string",
+      "description": "The output path of the generated files."
+    },
+    "devServerTarget": {
+      "type": "string",
+      "description": "The universal serve-ssr target that should be started."
+    },
+    "routesFile": {
+      "type": "string",
+      "description": "Path to the file that holds the route information."
+    },
+    "routes": {
+      "type": "string",
+      "description": "Comma separated list of routes."
+    }
+  }
```

Comment:
`outputPath` and `devServerTartet` are not marked optional in your interface, you should mark them as requiredAttributes here. 
routesFile and routes are marked as optional but do not define a default value here. Maybe add a description that either the routesFile or the routes are necessary

Reply:
the required was not working so I removed it. `routesFile` and routes are optional and have a fallback in the code. You cannot provide default values for it.

![image](https://user-images.githubusercontent.com/11156362/77630395-3cb5d000-6f4b-11ea-9aae-0ba5af43ee84.png)

![image](https://user-images.githubusercontent.com/11156362/77630421-48a19200-6f4b-11ea-9a6f-09bba9823f2b.png)